scripts/viz/param_sweep_viz.R

```diff
@@ -0,0 +1,74 @@
+# Pan-Cancer Variational Autoencoder
+# Gregory Way 2017
+#
+# Visualizing results of parameter sweep
+
+library(readr)
+library(ggplot2)
```

Comment:
me too! There's nothing in the style guide about it

It definitely makes sense to do if there are functions of the same name in different packages, and also if you only need single functions in a large library. But, given the simplicity of this script, I think I will stay with the call to `library(ggplot2)`. I will however, remove the calls to `library(dplyr)` and `library(readr)`. Change in next commit!